code/scripting/api/objs/player.cpp

```diff
@@ -266,16 +266,11 @@ ADE_FUNC(loadCampaignSavefile, l_Player, "string campaign = <current>", "Loads t
 scoring_stats_h::scoring_stats_h() = default;
 scoring_stats_h::scoring_stats_h(const scoring_struct& stats)
 {
-	_score = new scoring_struct();
```

Comment:
I would prefer the scoring changes to be in a separate PR.  What is the rationale?

Reply:
I think I introduced this suboptimal version in an earlier, related pull request and wanted to improve it here. Since it's already here and there is no real advantage in moving it to another PR I would like to keep this in here.